src/goto-programs/goto_model.h

```diff
@@ -142,12 +151,20 @@ class wrapper_goto_modelt : public abstract_goto_modelt
   ///
   /// The validation mode indicates whether well-formedness check failures are
   /// reported via DATA_INVARIANT violations or exceptions.
-  void validate(const validation_modet vm) const override
+  void validate(
+    const validation_modet vm,
+    const goto_model_validation_optionst &goto_model_validation_options)
+    const override
   {
     symbol_table.validate(vm);
 
+    // Does a number of checks at the function_mapt level to ensure the
+    // goto_program is well formed. Does not call any validate methods
+    // (at the goto_functiont level or below)
```

Comment:
:pick: ... especially if you need to explain it twice?

Reply:
Hmm, the name is the same as the command line parameter "validate_goto_model".
I don't see how to condense this and leaving it out may cause confusion "why is this here?"
It does not harm, may help. 
see also https://github.com/diffblue/cbmc/pull/3191#discussion_r257253294